src/ts/content/extensions/author/authorPage/authorUI.ts

```diff
@@ -82,7 +82,7 @@ const currentTagViewButtons = ({onPush, onSync, onEdit}: ButtonHandlers, getTags
 	);
 	const onEditButton = createTagButton("Edit", "img/edit.gif", onEdit);
 
-	onLogged({
+	void onLogged({
```

Comment:
i read the docs and i still don't understand what this does lol

Reply:
it casts the return value of the function call to `void`

Useful if you want to communicate "no i didn't forget to await this call -- this was on purpose that the promise is floating"